src/lib/mina/graphql.ts

```diff
@@ -250,22 +250,24 @@ type EventQueryResponse = {
   }[];
 };
 
-type ActionQueryResponse = {
-  actions: {
-    blockInfo: {
-      distanceFromMaxBlockHeight: number;
-    };
-    actionState: {
-      actionStateOne: string;
-      actionStateTwo: string;
-    };
-    actionData: {
-      accountUpdateId: string;
-      data: string[];
-    }[];
+export type FetchedAction = {
```

Comment:
nit: we usually do all exports at the top of a file, so we see what a file exports immediately